src/components/top-bar-equipment-seach-dialog/custom-suffix-renderer.tsx

```diff
@@ -48,11 +47,10 @@ export const CustomSuffixRenderer: FunctionComponent<
         (state: ReduxState) => state.networkAreaDiagramDepth
     );
 
-    const centerOnSubstationCB = useCallback(
-        (
-            e: React.MouseEvent<HTMLButtonElement, MouseEvent>,
-            element: EquipmentInfos
-        ) => {
+    const centerOnSubstationCB = useCallback<
+        MouseEventHandler<HTMLButtonElement>
+    >(
+        (e) => {
```

Comment:
```suggestion
    const centerOnSubstationCB = useCallback(
        (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
```
I have no type checking error with the code above